Content.Server/Disease/DiseaseZombieSystem.cs

```diff
@@ -0,0 +1,188 @@
+            base.Initialize();
+
+            SubscribeLocalEvent<DiseaseZombieComponent, ComponentInit>(OnComponentInit);
+            SubscribeLocalEvent<DiseaseZombieComponent, MeleeHitEvent>(OnMeleeHit);
+            SubscribeLocalEvent<DiseaseZombieComponent, RefreshMovementSpeedModifiersEvent>(OnRefreshMovementSpeedModifiers);
+        }
+        
+        /// <remarks>
+        /// I would imagine that if this component got assigned to something other than a mob, it would throw hella errors.
+        /// </remarks>
+        private void OnComponentInit(EntityUid uid, DiseaseZombieComponent component, ComponentInit args)
+        {
+            if (!component.ApplyZombieTraits || !HasComp<MobStateComponent>(uid))
+                return;
+
+            if (HasComp<DiseaseCarrierComponent>(uid))
+                EntityManager.RemoveComponent<DiseaseCarrierComponent>(uid);
+            if (HasComp<RespiratorComponent>(uid))
+                EntityManager.RemoveComponent<RespiratorComponent>(uid);
+            if (HasComp<BarotraumaComponent>(uid))
+                EntityManager.RemoveComponent<BarotraumaComponent>(uid);
+            if (HasComp<HungerComponent>(uid))
+                EntityManager.RemoveComponent<HungerComponent>(uid);
+            if (HasComp<ThirstComponent>(uid))
+                EntityManager.RemoveComponent<ThirstComponent>(uid);
+
+            EntityManager.EnsureComponent<BloodstreamComponent>(uid, out var bloodstream); //zoms need bloodstream anyway for healing
+            _bloodstream.SetBloodLossThreshold(uid, 0f, bloodstream);
+            _bloodstream.TryModifyBleedAmount(uid, -bloodstream.BleedAmount, bloodstream);
+            _movementSpeedModifier.RefreshMovementSpeedModifiers(uid);
+            EntityManager.EnsureComponent<ReplacementAccentComponent>(uid).Accent = "zombie";
+
+            if (TryComp<DamageableComponent>(uid, out var comp))
+            {
+                _damageable.SetDamageModifierSetId(uid, "Zombie", comp);
+                _damageable.SetAllDamage(comp, 0);
+            }
+
+            if(TryComp<HumanoidAppearanceComponent>(uid, out var spritecomp))
+            {
+                //zombie hex color is #aeb87bff
+                var color = new Color(0.70f, 0.72f, 0.48f, 1);
+
+                var oldapp = spritecomp.Appearance;
+                var newapp = oldapp.WithSkinColor(color);
+                _sharedHumanoidAppearance.UpdateAppearance(uid, newapp);
+				
+				_sharedHumanoidAppearance.ForceAppearanceUpdate(uid);
+            }
+
+            if(TryComp<HandsComponent>(uid, out var handcomp))
+            {
+                foreach (var hand in handcomp.Hands)
+                {
+                    _sharedHands.TrySetActiveHand(uid, hand.Key);
+                    _sharedHands.TryDrop(uid);
+
+                    var pos = EntityManager.GetComponent<TransformComponent>(uid).Coordinates;
+                    var virtualItem = EntityManager.SpawnEntity("ZombieClaw", pos);
```

Comment:
```suggestion
                    var claw = EntityManager.SpawnEntity("ZombieClaw", pos);
```

this isn't a virtual item